src/docs/sphinx/tutorials/step11/Tutorial.rst

```diff
@@ -0,0 +1,234 @@
+.. _TutorialPlasticity:
+
+
+####################################################
+Tutorial 11: Plasticity Model for Wellbore Problems
+####################################################
+
+
+**Context**
+
+The main goal of this tutorial is to learn how to use the internal mesh generator and plasticity models to handle wellbore problems in GEOSX. The Extended Drucker-Prager model (see :ref:`TwoInvariantPlasticity`) is applied to solve for elastoplastic deformation within the vicinity of a vertical wellbore. For the presented example, an analytical solution is employed to verify the accuracy of the numerical results. The resulting model can be used as a base for more complex analysis (e.g., wellbore drilling, fluid injection and storage scenarios). 
+
+
+**Objectives**
+
+At the end of this tutorial you will know:
+
+  - how to construct meshes for wellbore problems with the internal mesh generator,
+  - how to specify initial and boundary conditions, such as in-situ stresses and variation of traction at the wellbore wall,
+  - how to use a plastic model for mechanical problems in the near wellbore region.
+
+
+**Input file**
+
+This tutorial uses no external input files and everything required is
+contained within a single GEOSX input file. 
+The xml input file for this test case is located at:
+
+.. code-block:: console
+
+  examples/plasticity/DruckerPrager_Verification.xml
+
+
+------------------------------------------------------------------
+Description of the case
+------------------------------------------------------------------
+
+We simulate a drained wellbore problem subjected to isotropic horizontal stress (:math:`\Sigma_h`) and vertical stress (:math:`\Sigma_v`). By lowering the wellbore supporting pressure (:math:`P_w`), the wellbore contracts, and the reservoir rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region, as shown in Fig.1.
+
+
+.. _problemSketchFig:
+.. figure:: Wellbore.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Fig.1 Sketch of the wellbore problem `(Chen and Abousleiman, 2017)  <https://www.sciencedirect.com/science/article/pii/S1365160917301090>`__
+
+
+To simulate this phenomenon, the strain hardening Extended Drucker Prager model with an associated flow rule in GEOSX is used in this tutorial. Displacement and stress fields around the wellbore are numerically calculated. These numerical predictions are then compared with the corresponding analytical solutions `(Chen and Abousleiman, 2017)  <https://www.sciencedirect.com/science/article/pii/S1365160917301090>`__ from the literature. 
```

Comment:
```suggestion
To simulate this phenomenon, the strain hardening Extended Drucker-Prager model with an associated flow rule in GEOSX is used in this tutorial. Displacement and stress fields around the wellbore are numerically calculated. These numerical predictions are then compared with the corresponding analytical solutions `(Chen and Abousleiman, 2017)  <https://www.sciencedirect.com/science/article/pii/S1365160917301090>`__ from the literature. 
```

Reply:
This is the flow rule for plasticity model, not the fluid flow.

Reply:
You may add "plastic flow" to avoid confusion

Reply:
agree